cnxcommon/urlslug.py

```diff
@@ -64,6 +64,7 @@ def remove_html_tags(title):
 
 @utils.ensure_unicode
 def get_os_number(title):
-    m = re.search('<span class="os-number">([^<]+)</span>', title)
+    tmp_title = re.sub(r'<span class="os-part-text">([^<]+)</span>', "", title)
```

Comment:
Super minor, but since we have the same regex in two places now in `get_os_number` and `remove_html_tags`, should we define that as a constant or a shared function?

Reply:
:blush: fixed